static/src/js/components/AppLogo/AppLogo.js

```diff
@@ -1,107 +1,55 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {
+  Link,
   Route,
   Switch
 } from 'react-router-dom'
 
 import PortalLinkContainer from '../../containers/PortalLinkContainer/PortalLinkContainer'
 
-import { isDefaultPortal } from '../../util/portals'
-
 import './AppLogo.scss'
 
-const AppLogo = ({
-  portal
-}) => {
-  const {
-    portalId,
-    logo = {},
-    org,
-    title = {}
-  } = portal
-
-  const { primary: primaryTitle } = title
-
-  let portalLink = '/'
-  if (!isDefaultPortal(portalId)) {
-    portalLink = `/portal/${portalId}/search`
-  }
-
-  const portalLogo = () => {
-    if (!Object.keys(logo).length) return null
-
-    const {
-      id,
-      image,
-      link = portalLink,
-      title: logoTitle
-    } = logo
-
-    return (
-      <a
-        id={id}
-        className="app-logo__portal-logo"
-        href={link}
-        title={logoTitle}
-      >
-        {
-          image && (
-            <img src={image} alt={logoTitle} />
-          )
-        }
-        <span className="visually-hidden">
-          {logoTitle}
-        </span>
-      </a>
-    )
-  }
-
-  return (
-    <nav className="app-logo">
-      <span className="app-logo__site-logo">
-        <h1 className="mb-0">
-          <a
-            className="app-logo__site-meatball"
-            href="/"
-            title="Earthdata Search Home"
-          >
-            <span className="visually-hidden">
-              Earthdata Search
-            </span>
-          </a>
-        </h1>
-        {portalLogo()}
+const AppLogo = () => (
+  <nav className="app-logo">
+    <span className="app-logo__site-logo">
+      <h1 className="mb-0">
         <a
-          className="app-logo__site-name"
-          href={portalLink}
+          className="app-logo__site-meatball"
+          href="/"
+          title="Earthdata Search Home"
         >
-          <span className="app-logo__site-name-ent app-logo__site-name-ent--e">{org}</span>
-          <span className="app-logo__site-name-ent app-logo__site-name-ent--s">{primaryTitle}</span>
+          <span className="visually-hidden">
+            Earthdata Search
+          </span>
         </a>
-      </span>
-      <Switch>
-        <Route path="/admin">
-          <PortalLinkContainer
-            className="app-logo__header-site-area-link"
-            to="/admin"
-          >
-            <h2 className="app-logo__header-site-area-title">Admin</h2>
-          </PortalLinkContainer>
-        </Route>
-      </Switch>
-    </nav>
-  )
-}
+      </h1>
+      <Link
```

Comment:
I think @macrouch expressed some hesitation around using a `Link` for this. IMO, it is a UX improvement to use the `Link` and we should get to a place where we feel confident the link is resetting things, and remember that we can always just refresh our browser to truly "reset" the page :)

Either way, now that we are not showing the portal title in this spot, I think we can have a single `a` or `Link` wrapping both the meatball and the site header text rather than have them as separate elements. It feels like this could be done after things get merged. We just need to make sure we dont lose sight of it.